cmd/skopeo/layers.go

```diff
@@ -19,8 +19,25 @@ var layersCmd = cli.Command{
 	Name:      "layers",
 	Usage:     "Get layers of IMAGE-NAME",
 	ArgsUsage: "IMAGE-NAME [LAYER...]",
+	Hidden:    true,
```

Comment:
This would be worth calling out… if not a separate PR, at least a separate commit?